packages/fast-usdc/src/exos/transaction-feed.js

```diff
@@ -36,6 +36,22 @@ const TransactionFeedKitI = harden({
   }),
 });
 
+/**
+ * @param {globalThis.MapStore<string, RiskAssessment>[]} riskStores
```

Comment:
nit: use the type instead of the value. jsdoc is sloppy with the distinction but really this is `typeof globalThis.MapStore` which could just be `MapStore` if you imported that type